src/TiledArray/dist_array.h

```diff
@@ -1595,7 +1595,7 @@ inline std::ostream& operator<<(std::ostream& os,
                                 const DistArray<Tile, Policy>& a) {
   if (a.world().rank() == 0) {
     for (std::size_t i = 0; i < a.size(); ++i)
-      if (!a.is_zero(i)) {
+      if (a.is_local(i) && !a.is_zero(i)) {
```

Comment:
this will cause only tiles resident on node 0 to be printed.

What is the problem you are trying to solve?

Reply:
> this will cause only tiles resident on node 0 to be printed

This is how the function is described in line 1585. I'd prefer this function to print all tiles indeed by removing the rank check at line 1596. However, I still don't know why this function is giving a seq fault (without these changes) while it was working before. I find about this while trying to understand a seg fault in LibChemist unit tests.

Reply:
I think I got it now, previously tensors had replicated pmap by default and this function was written for them. I added 53b1e15e804b211a9202209706ef033b6d7d8c07 to enable printing all tiles.

Reply:
We probably don't want to rely on MPI to transport output from other nodes, it's better to print from node 0 directly (i.e. using madness to transport data and then print locally) ... what's the problem with printing replicated tensors?

Reply:
No problem with replicated tensors, it only seg faults with blocked or cyclic tensors